extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php

```diff
@@ -20,6 +20,16 @@ public function __construct() {
 	}
 
 	/**
+	 * Check if a user is an admin
+	 *
+	 * @param string $group The user's group from ug_group
+	 * @return bool
+	 */
+	public function isAdmin( $group ) {
+		return strpos( $group, 'sysop' ) !== false;
```

Comment:
Well, you can create the user object and get all groups, but doing this for every user would be potentially expensive. Have you considered using `WikiService::getWikiAdminIds`, which should only do one efficient query if it's not in cache, and then just use `in_array` for `isAdmin`?